`#include <iostream>` missing from _OutputSoundFile.cpp_

### Prerequisite Checklist

- [x] I searched for [existing issues](https://github.com/search?q=repo%3ASFML%2FSFML&type=issues) to prevent duplicates
- [x] I searched for [existing discussions on the forum](https://www.google.com/search?q=site%3Ahttps%3A%2F%2Fen.sfml-dev.org) to prevent duplicates
- [x] I am here to report an issue and not to just ask a question or look for help (use the [forum](https://en.sfml-dev.org/forums/index.php#c3) or [Discord](https://discord.gg/nr4X7Fh) instead)

### Describe your issue here

SFML doesn't compile with Clang 20
```
src/SFML/Audio/OutputSoundFile.cpp:71:15: error: invalid operands to binary expression ('std::ostream' (aka 'basic_ostream<char>') and 'const char[65]')
   71 |         err() << "Failed to open output sound file from file (writer open failure)" << std::endl;
```

### Your Environment

- OS / distro / window manager: Windows, MSYS2, clang64
- SFML version: 3.0.1
- Compiler / toolchain: Clang 20
- Special compiler / CMake flags: -stdlib=libc++ (-std=c++26 - probably not important)

### Steps to reproduce

- Compile SFML as a library

### Expected behavior

No compilation error

### Actual behavior

```
src/SFML/Audio/OutputSoundFile.cpp:71:15: error: invalid operands to binary expression ('std::ostream' (aka 'basic_ostream<char>') and 'const char[65]')
   71 |         err() << "Failed to open output sound file from file (writer open failure)" << std::endl;
```

--- a/src/SFML/Audio/OutputSoundFile.cpp
+++ b/src/SFML/Audio/OutputSoundFile.cpp
@@ -32,6 +32,8 @@
 #include <SFML/System/Err.hpp>
 #include <SFML/System/Exception.hpp>
 
+#include <ostream>
+
 #include <cassert>
 
 

